x-pack/plugins/security_solution/public/management/pages/event_filters/store/utils.ts

```diff
@@ -10,6 +10,14 @@ import type { CreateExceptionListItemSchema } from '@kbn/securitysolution-io-ts-
 import { Ecs } from '../../../../../common/ecs';
 import { ENDPOINT_EVENT_FILTERS_LIST_ID } from '../constants';
 
+const osTypeBasedOnAgentType = (data?: Ecs) => {
```

Comment:
Please disregard if this is not relevant:

Here we assume the OS to be windows. Is this reliable? are we sure that all events in this event list come from windows hosts?

Reply:
Not exactly. `Windows` will be the default value if for some reason host.os.name or host.os.family are empty, otherwise we will use the host provided by the event data.